app/scripts/museum/museum-service.js

```diff
@@ -100,6 +106,30 @@
             return Util.makeRequest(sql, query);
         }
 
+        function listByCity(params) {
```

Comment:
Have some time left on this task, would be interested in ideas for ways to make this a bit better, although it works well enough as is.